app/experimenter/nimbus-ui/src/components/PageNew/index.tsx

```diff
@@ -47,20 +50,33 @@ const PageNew: React.FunctionComponent<PageNewProps> = () => {
         const { message, nimbusExperiment } = result.data.createExperiment;
 
         if (message !== "success" && typeof message === "object") {
+          setIsServerValid(false);
           return void setSubmitErrors(message);
+        } else {
+          setIsServerValid(true);
+          setSubmitErrors({});
         }
         resetForm();
         navigate(`${nimbusExperiment!.slug}/edit/overview`);
       } catch (error) {
-        setSubmitErrors({ "*": SUBMIT_ERROR });
+        setSubmitErrors({ "*": `${SUBMIT_ERROR}` });
       }
     },
     [createExperiment],
   );
 
   return (
     <AppLayout testid="PageNew">
-      <h1 className="h2">Create a new Experiment</h1>
+      <Row>
+        <Col>
+          <h1 className="h2">Create a new Experiment</h1>
+        </Col>
+        <Col className="text-right">
+          <button className="btn pr-0" onClick={onFormCancel}>
+            &#10005;
```

Comment:
Can we use the SVG glyph here? Looks like we already have it in the repo, so just

```tsx
import { ReactComponent as DeleteIcon } from "../../images/x.svg";

// and then
<DeleteIcon width="18" height="18" />
```